docs/Single Sign On/okta.md

```diff
@@ -17,21 +18,57 @@ Login to your Okta account as an admin and navigate to `Applications -> Applicat
 
 In the dialog that opens, choose **OIDC - Open ID Connect** as the **Sign-in Method** and **Web Application** as the **Application Type.**
 
-On the next screen provide a name for the application. Under **Assignments -> Controlled Access**, select `Skip Group Assignment For Now`
-
 <img src="/img/docs/oidc-okta-1.png" alt="Creating New Okta OAuth App" height="75%" width="75%"/>
 
 Leave the **Application Callback URL** empty for now and Register the application.
 
-Note down the `client-id` and `client-secret` as those will be required to configure identity provider for Paralus.
+Based on who can access the Paralus app integration, select **Controlled access** options.
+
+Note down the `Client ID` and `Client Secret` as those will be required to configure identity provider for Paralus.
+
+### Add a custom claim for groups mapping
+
+Paralus comes with the in-built functionality for mapping Identity
+provider (IdP) groups to Paralus. This is useful to manage different
```

Comment:
`This is useful to manage different Paralus access to the IdP users through IdP groups they are part of. It eases the access-management process for Parlaus administrators`

We can rephrase this to "_This makes access management simpler. As an administrator, you don't need to manage access for an individual user but for a group of users_"